README.md

```diff
@@ -1,5 +1,14 @@
+# GroestlCoin Wallet
+
 Welcome to _GroestlCoin Wallet_, a standalone GroestlCoin payment app for your Android device!
 
+[<img src="https://fdroid.gitlab.io/artwork/badge/get-it-on.png"
+     alt="Get it on F-Droid"
+     height="80">](https://f-droid.org/packages/hashengineering.groestlcoin.wallet/)
+[<img src="https://play.google.com/intl/en_us/badges/images/generic/en-play-badge.png"
+     alt="Get it on Google Play"
+     height="80">](https://play.google.com/store/apps/details?id=hashengineering.groestlcoin.wallet)
+
```

Comment:
There is also a testnet app on Google Play:  https://play.google.com/store/apps/details?id=hashengineering.groestlcoin.wallet_test

The testnet app is not available on Android.

Most may not care about the testnet app, so perhaps we don't need links with images on the top of the page, but the bottom of the page could remain the same as before, but with the correct links.